_posts/2020-01-21-hhvm-4.41.markdown

```diff
@@ -0,0 +1,40 @@
+---
+title: "HHVM 4.41"
+layout: post
+author: jjergus
+category: blog
+---
+
+HHVM 4.41 is released! This release marks the end of support for 4.34;
+HHVM 4.35&ndash;4.40 remain supported, as do the 4.8 and 4.32 LTS releases.
+
+# Highlights
+
+- A new type alias, `HH\enumname<TEnum>`, represents the name of the enum type
+  `TEnum` (similarly to `classname<TClass>`).
+- A new function, `HH\type_structure_for_alias`, returns the `TypeStructure` of
+  the specified type alias (similarly to `HH\type_structure` for type constants
+  in classes).
+  - Note that the `TypeStructure` API remains experimental and may be changed or
+    removed in the future.
+- Builtin functions `usort`, `uasort`, `uksort` now accept callbacks that return
+  any `num` value (`int` or `float`), allowing for more convenient sorting of
+  `float` arrays.
```

Comment:
not sure if worth mentioning, since everybody should prefer HSL

Reply:
I think it is, but:
- is it a breaking change?
- call out that the HSL is strongly preferred

Reply:
Not breaking, the functions now accept more callbacks than before.

Reply:
I think the changelog here is definitely valuable, even if users should prefer HSL.